docs/pages/eas-update/asset-selection.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Asset selection and exclusion
+description: Select which assets should be included in updates.
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { GithubIcon } from '@expo/styleguide-icons';
+
+> **info** Available for SDK 50 and above ([`expo-updates`](/versions/latest/sdk/updates/) >= 0.23.0).
+
+In SDK 49 and earlier, all assets resolved in the Metro bundler are included in every update, and are uploaded to the update server. The new experimental asset selection feature in SDK 50 allows the developer to specify that only certain assets should be included in updates. This has the potential to greatly reduce the number of assets that need to be uploaded to and downloaded from the updates server. This feature will work with the EAS update server, or any custom server that complies with the [`expo-updates` protocol](/technical-specs/expo-updates-1.mdx).
```

Comment:
```suggestion
In SDK 49 and earlier, all assets resolved in the Metro bundler are included in every update, and are uploaded to the update server. The new experimental asset selection feature in SDK 50 allows the developer to specify that only certain assets should be included in updates. This can greatly reduce the number of assets that need to be uploaded to and downloaded from the updates server. This feature will work with the EAS update server or any custom server that complies with the [`expo-updates` protocol](/technical-specs/expo-updates-1).
```

- We can be a bit concise here
- Also, in internal links, we can avoid mentioning file extensions